3p/integration.js

```diff
@@ -523,6 +524,7 @@ register('rubicon', rubicon);
 register('runative', runative);
 register('sas', sas);
 register('seedingalliance', seedingalliance);
+register('seedtag', seedtag);
```

Comment:
Change to this file is no longer needed. Please revert the change to `3p/integration.js`.